oauth-keycloak-authorizer/etc/authorization-scopes.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
Could you please explain why do we need this file here? How is it used and how is it packaged?

Reply:
Currently all the documentation is in [examples/docker/README-authz.md](https://github.com/mstruk/strimzi-kafka-oauth/blob/authz/examples/README-authz.md) which is a walkthrough example. This file is mentioned there. I can add a paragraph that explains how to import it into empty Authorization Services configuration.

Reply:
Fixed.